### PR TITLE
Update recipe for 0.3.2, now v0.3.3

### PR DESCRIPTION
**Pull request by kkappler**
- Update version to 0.3.2 and replace sha256
- update to python >=3.8
- add missing dependencies
- remove aurora.signal from test

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by kkappler** [comment]
@conda-forge-admin, please rerender

**Comment by kkappler** [comment]
@lheagy This PR looks ready to review as the re-rendering is now complete

**Review by lheagy — APPROVED** [review]

**Comment by kkappler** [comment]
@conda-forge-admin 

I am a maintainer, listed in the meta.yaml for this repositiory, but I was unable to merge my PR even after it was approved by another maintainer.  Normally I see a Merge button but this seems not available to me.  Am I missing something?

image

**Pull request merged**
